packages/playwright-ct-vue/index.d.ts

```diff
@@ -25,6 +25,7 @@ import type {
 } from '@playwright/test';
 import type { JsonObject } from '@playwright/experimental-ct-core/types/component';
 import type { InlineConfig } from 'vite';
+import type { ComponentProps } from 'vue-component-type-helpers';
```

Comment:
It is hosted in the Vue Github organization: https://github.com/vuejs/language-tools.

I think the `componentProps<>` could change later on as some component definitions return any instead of the actual prop type. Updating a package makes it easier to deal with that.

Would you like me to copy the type or keep it as is? I'm fine with both.